README.md

```diff
@@ -1,8 +1,16 @@
-# Installation
-                          
-> ⚠️ Please use the Grafana Agent configuration for production use cases. 
+# Overview
+
+The grafana-opentelemetry-starter makes it easy to use Metrics, Traces, and Logs with OpenTelemetry 
+in Grafana Cloud or with Grafana Agent (for Grafana Cloud or Grafana OSS stack). 
+
+# Compatibility
 
-Spring boot starter to use Metrics, Traces, and Logs with OpenTelemetry in Grafana Cloud or with Grafana Agent (for Grafana Cloud or Grafana OSS stack) 
+| Spring Boot Version | Recommended Setup                                                                        |
+|---------------------|------------------------------------------------------------------------------------------|
+| 3.0.4+              | Use this starter                                                                         |
+| 2.x                 | Use the [Java Agent](https://grafana.com/docs/opentelemetry/instrumentation/java-agent/) |
+
+# Installation
```

Comment:
The yaml blocks suggest this, but I think the README should state that users should set either cloud or onprem properties but not both.